metabase/metabase
# Show inactive tables in admin

Epic: https://github.com/metabase/metabase/issues/18675

Adds disabled tables to Admin pages. These tables have not yet been synced by Metabase.

**Data Model -> Select database**

<img width="293" alt="Screen Shot 2021-10-28 at 5 20 29 pm" src="https://user-images.githubusercontent.com/8542534/139275309-18e811d5-592b-4ad9-b27f-cd46ee2f5570.png">

**Databases -> List**

<img width="330" alt="Screen Shot 2021-10-28 at 5 20 39 pm" src="https://user-images.githubusercontent.com/8542534/139275315-0884da67-aeca-4f26-a789-e75304698fd1.png">

**Databases -> Select database**

<img width="453" alt="Screen Shot 2021-10-28 at 5 20 47 pm" src="https://user-images.githubusercontent.com/8542534/139275318-816414cb-2646-48f1-95ad-113d9bb8dfe8.png">

How to test:
As the BE for this PR is not ready we would need to rely on mocked data:
1. Set `initial_sync` to false https://github.com/metabase/metabase/blob/c0b5c697d1c5c814ec35a50ed2696617ea172d4a/frontend/src/metabase/schema.js#L19
2. Set `initial_sync` to false https://github.com/metabase/metabase/blob/c0b5c697d1c5c814ec35a50ed2696617ea172d4a/frontend/src/metabase/schema.js#L53
3. Navigate to `http://localhost:3000/admin/databases`. Please check all databases are selectable but have loading indicators.
4. Navigate to a specific database, e.g. `http://localhost:3000/admin/databases/1`. Please check that sync buttons are disabled.
5. Navigate to Data model section for a database, e.g. `http://localhost:3000/admin/datamodel/database/1`. Please check that tables are disabled.
6. Revert changes to `initial_sync` flag and verify that all databases/tables can be selected.

Please note that it should be OK to merge this PR before BE changes as with `initial_sync` set statically to `true` users won't see FE changes.
